### PR TITLE
conway governance tasks

### PR DESCRIPTION
**Pull request by ecioppettini**
~For the most part this can be reviewed, although I probably will do some renames and small changes. **BUT** the tasks are not tested yet. Mostly because current version of cml parsing fails at an early block in sancho. I have tested the endpoints with mocked data in the tables, and also tested that this doesn't break in preview, but ideally I'd like to test the tasks with actual certificates.~

### CODE REVIEW

**Comment by rooooooooob** [comment]
>current version of cml parsing fails at an early block in sancho

@ecioppettini hopefully dcSpark/cardano-multiplatform-lib#324 will fix it. I'll test out syncing sancho later after I update multi-era (soon).

File: indexer/tasks/src/multiera/multiera_address.rs [hunk 1]
```diff
@@ -172,8 +182,18 @@ fn queue_certificate(
...
+
+            vkey_relation_map.add_relation(
+                tx_id,
+                &credential,
+                TxCredentialRelationValue::StakeDeregistration,
+            );
+        }
         MultiEraCertificate::PoolRegistration(registration) => {
-            let operator_credential = registration.pool_params.operator.to_raw_bytes().to_vec();
+            let operator_credential =
```
Comment (ecioppettini) [inline indexer/tasks/src/multiera/multiera_address.rs:195]: this is unrelated, but I think this was incorrect, and it was incorrectly ported in the pallas to cml migration.

**Review by SebastienGllmt — APPROVED** [review]

File: Cargo.toml [hunk 2]
```diff
@@ -9,3 +9,10 @@ members = [
     "indexer/plan-visualizer",
     "indexer/task-docgen"
 ]
+
+
+[workspace.dependencies]
+cml-chain = { git = "https://github.com/dcSpark/cardano-multiplatform-lib", branch = "new-conway-cddl-updates" }
+cml-core = { git = "https://github.com/dcSpark/cardano-multiplatform-lib", branch = "new-conway-cddl-updates" }
+cml-crypto = { git = "https://github.com/dcSpark/cardano-multiplatform-lib", branch = "new-conway-cddl-updates" }
+cml-multi-era = { git = "https://github.com/dcSpark/cardano-multiplatform-lib", branch = "new-conway-cddl-updates" }
```
Comment (SebastienGllmt) [inline Cargo.toml:18]: I need to fix this before merging

File: indexer/tasks/src/multiera/multiera_drep_delegation.rs [hunk 3]
```diff
@@ -0,0 +1,130 @@
...
+carp_task! {
+  name MultieraDrepDelegationTask;
+  configuration EmptyConfig;
+  doc "Tracks stake delegation actions to dreps";
+  era multiera;
+  dependencies [MultieraStakeCredentialTask];
+  read [multiera_txs, multiera_stake_credential];
+  write [];
+  should_add_task |block, _properties| {
+    // recall: txs may have no outputs if they just burn all inputs as fee
```
Comment (SebastienGllmt) [inline indexer/tasks/src/multiera/multiera_drep_delegation.rs:34]: this comment here is probably a copy-paste mistake (the TODO is correct though)

File: indexer/tasks/src/multiera/multiera_governance_voting.rs [hunk 4]
```diff
@@ -0,0 +1,61 @@
...
+carp_task! {
+  name MultieraGovernanceVotingTask;
+  configuration EmptyConfig;
+  doc "";
+  era multiera;
+  dependencies [MultieraTransactionTask];
+  read [multiera_txs];
+  write [];
+  should_add_task |block, _properties| {
+    // recall: txs may have no outputs if they just burn all inputs as fee
```
Comment (SebastienGllmt) [inline indexer/tasks/src/multiera/multiera_governance_voting.rs:16]: same for the comment here

**Review by SebastienGllmt — APPROVED** [review]

**Pull request merged**